packages/adapter-cloudflare-workers/index.js

```diff
@@ -97,7 +97,10 @@ export default function ({ config = 'wrangler.toml', platformProxy = {} } = {})
 					alias: Object.fromEntries(compatible_node_modules.map((id) => [id, `node:${id}`])),
 					format: 'esm',
 					loader: {
-						'.wasm': 'copy'
+						'.wasm': 'copy',
```

Comment:
it looks like eot and otf are the other two coming up in my searches

Reply:
Alright, I've added `.eot` and `.otf` too. Just curious, what kind of search are you conducting?

Reply:
i'd done a good search for "esbuild loader font" or something like that